test/src/button-contributions.spec.ts

```diff
@@ -78,7 +78,7 @@ describe("Query Selector Tests", function () {
 
     async function resolveSelector(page: Page, selector: string) {
         if (selector.startsWith("xpath:")) {
-            return (await page.$x(selector.slice(6)))[0] || null;
```

Comment:
This just fixes an issue where depending on how quickly the typescript plugin initializes in VSCode, you can get a `Property '$x' does not exist on type 'Page'` error when running `pnpm test`